mne/gui/__init__.py

```diff
@@ -200,6 +200,8 @@ def coregistration(tabbed=False, split=True, width=None, inst=None,
     )
 
 
+@deprecated('Use the `mne-gui-addons` package instead, will be '
```

Comment:
```suggestion
@deprecated('Use the :mod:`mne-gui-addons` package instead, will be '
```
And you'll need to add mne-gui-addons to `doc/conf.py` intersphinx. Or remove the backticks, up to you. It will fix a CircleCI warning. (The others/pandas ones should be fixed by a re-run or new push anyway.)